composer.json

```diff
@@ -54,7 +54,7 @@
         "symfony/console": "^3.4 || ^4.0",
         "symfony/debug": "^2.8 || ^3.0 || ^4.0",
         "symfony/dependency-injection": "^3.4 || ^4.0",
-        "symfony/doctrine-bridge": "^2.8.12 || ^3.0 || ^4.0",
+        "symfony/doctrine-bridge": "^4.0",
```

Comment:
If it is supported by Symfony 4.0, it should also be supported by Symfony 3.4. We should at least support 3.4 because it's the latest LTS version.